src/libs/utils.ts

```diff
@@ -35,18 +37,12 @@ export const makePrettyDate = (dateString: number | string | Date): string => {
   return cond(
     [isToday(date), () => 'Today'],
     [isYesterday(date), () => 'Yesterday'],
-    [differenceInCalendarMonths(date, Date.now()) <= 6, () => dateFormat.format(date)],
+    [differenceInCalendarMonths(date, Date.now()) <= 6, () => formatDate(date)],
     () => monthYearFormat.format(date)
   );
 };
 
-export const makeStandardDate = (dateString: number | string | Date): string => dateFormat.format(new Date(dateString));
-
-export const makeCompleteDate = (dateString: number | string | Date): string =>
-  completeDateFormat.format(new Date(dateString));
-
-export const formatTimestampInSeconds = (secondsSinceEpoch: number): string =>
-  completeDateFormat.format(new Date(secondsSinceEpoch * 1000));
+export const formatTimestampInSeconds = (secondsSinceEpoch: number): string => formatDatetime(secondsSinceEpoch * 1000);
 
 export const makeCompleteDateParts = (dateString) => {
```

Comment:
These two functions are only used in one place each, so I think we can replace them.